autoload/go/term.vim

```diff
@@ -67,11 +66,7 @@ function! go#term#newmode(bang, cmd, mode) abort
   " we also need to resize the pty, so there you go...
   call jobresize(state.id, width, height)
 
-  stopinsert
-
-  if state.winnr !=# winnr()
-    exe state.winnr . "wincmd w"
-  endif
+  call win_gotoid(state.winid)
```

Comment:
is this a native nvim function?

Reply:
It is. We actually probably need to be using it many other places, but all the other cases where it should be used but isn't requires an unlikely edge case before a user would notice a problem. Because of the way the terminal works, though, we must use the window id instead of the window number to make sure the behavior is the same regardless of the value of `splitright`.

Reply:
Thanks for the clarification.